pkgs/tools/security/sbomnix/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, coreutils
+, curl
+, gnugrep
+, gnused
+, gzip
+, nix
+, python3Packages
```

```suggestion
, python
```
this fixes python version mixing

---

Fixed in 8fae5ee3213024a3f498ebccbc1e94ac4a77ef8e